docs/001_develop/02_server-capabilities/006_core-business-logic-event-handler/index.mdx

```diff
@@ -459,7 +459,48 @@ As with `onValidate` the block must finish with an [`ack`](#ack). If the event i
     }
   }
 
-``` 
+```
+
+##### `trigger`
+
+The `trigger` method can be used to call on another event within the same Event Handler script file. It takes the event metadata as a parameter.
+
+```kotlin
+    eventHandler<TriggerTrade>(name = "TRIGGER_TRADE_INSERT") {
+        onValidate {
+            ack()
+        }
+        onCommit { event ->
+            val trade = Trade {
+                // do some logic with event details to get new details for next event call
+            }
+            // set the messageType to the name of the event you wish to call and details to the new trade object
+            trigger(event.copyMeta(messageType = "TRADE_INSERT", details = trade))
+        }
+    }
+```
+
+##### `triggerRemote`
+
+The `triggerRemote` method can be used to call on an event from another Event Handler script file (including those in another microservice). It takes the following parameters:
```

Comment:
Sorry but can we remove the (including those in another microservice) part. I don't feel it is relevant usage is determined by file.

Reply:
done